guide.md

```diff
@@ -45,51 +45,159 @@ We designed this guide to help anyone navigate those complexities—regardless o
+
+{% include image.html
+   image = "/assets/images/guide/designing-bitcoin-products.png"
+   retina = "/assets/images/guide/designing-bitcoin-products@2x.png"
+   alt-text = ""
+   width = 100
+   height = 100
+   layout = "float-left"
+%}
+
+<h2 class="h3" markdown="1">[Designing bitcoin products]({{ '/guide/designing-products/introduction/' | relative_url }})</h2>
 
-This section is an in-depth design exploration and reference design of a daily spending bitcoin wallet.
+A closer look at the design process and frameworks, from [personal finance use cases]({{ '/guide/designing-products/personal-finance/' | relative_url }}) to [units & symbols]({{ '/guide/payments/units-and-symbols/' | relative_url }}), the [usage life cycle]({{ '/guide/designing-products/usage-life-cycle/' | relative_url }}), [user research]({{ '/guide/designing-products/user-research/' | relative_url }}), and more.
+
+</div>
 
 ---
 
-### [Case studies]({{ '/guide/case-studies/' | relative_url }})
+<h4 class="h3" markdown="1">Reference designs</h4>
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending.png"
+   retina = "/assets/images/guide/daily-spending@2x.png"
+   imageMobile = "/assets/images/guide/daily-spending-mobile.png"
+   retinaMobile = "/assets/images/guide/daily-spending-mobile@2x.png"
+   alt-text = "Strong handshake"
+   width = 1600
+   height = 840
+%}
+
+<h2 class="h3" markdown="1">[Daily spending wallet]({{ '/guide/daily-spending-wallet/' | relative_url }})</h2>
+
+An in-depth exploration of a mobile wallet for a Lightning-first, on-the-go use case. Covers primary user flows like [first use]({{ '/guide/onboarding/first-use/' | relative_url }}), [sending]({{ '/guide/payments/send/' | relative_url }}) and [requesting]({{ '/guide/payments/request/' | relative_url }}), features like [backup]({{ '/guide/onboarding/backing-up-a-wallet/' | relative_url }}) and [contacts]({{ '/guide/payments/contacts/' | relative_url }}), and more.
+
+<div class="columns-desktop -two -ref">
+<div class="column" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/savings-account.png"
+   retina = "/assets/images/guide/savings-account@2x.png"
+   alt-text = "Strong handshake"
+   width = 400
+   height = 210
+%}
+
+<h2 class="h3" markdown="1">[Savings wallet]({{ '/guide/case-studies/savings-account/' | relative_url }})</h2>
+
+A multi-key mobile wallet design that provides great security for larger amounts, and low-friction for smaller transactions.
+
+</div>
+<div class="column" markdown="1">
 
-Explores approaches to diverse uses cases with wireframes, designs and prototypes.
+{% include picture.html
+   image = "/assets/images/guide/upgradeable-account.png"
+   retina = "/assets/images/guide/upgradeable-account@2x.png"
+   alt-text = "Strong handshake"
+   width = 400
+   height = 210
+%}
+
+<h2 class="h3" markdown="1">[Upgradeable account]({{ '/guide/case-studies/upgradeable-wallet/' | relative_url }})</h2>
```

Comment:
We should call this upgradeable wallet to fit the other reference designs.

Reply:
Separate PR.